pwa-devdocs/src/_includes/layout/pre-release-banner.html

```diff
@@ -1,4 +1,4 @@
 <div class="message-banner">
   The content on this site reflects the current progress of the Magento PWA project.
-  If you have any questions, concerns, or contribution ideas, join our <a href="https://magentocommeng.slack.com/messages/C71HNKYS2"><strong>#pwa</strong></a> Slack channel.
+  If you have any questions, concerns, or contribution ideas, join our <a href="https://magentocommeng.slack.com/messages/C71HNKYS2"><strong>#pwa</strong></a> Slack channel. If you are not registered yet, find out how to create an account in our [Community Resources](https://devdocs.magento.com/community/resources/resources.html).
```

Comment:
The link should be in HTML instead of markdown. I've also recommended a rewording to keep the content short.

```suggestion
If you have any project questions, concerns, or contribution ideas, join our <a href="https://magentocommeng.slack.com/messages/C71HNKYS2"><strong>#pwa</strong></a> Slack channel.
Find out how to create an account by visiting <a href="https://devdocs.magento.com/community/resources/resources.html">Community Resources</a>
```